packages/app-builder-lib/src/node-module-collector/npmNodeModulesCollector.ts

```diff
@@ -0,0 +1,25 @@
+import { execSync } from "child_process"
+import { NodeModulesCollector } from "./nodeModulesCollector"
+import { DependencyTree } from "./types"
+
+export class NpmNodeModulesCollector extends NodeModulesCollector {
+  constructor(rootDir: string) {
+    super(rootDir)
+  }
+
+  getPMCommand(): string {
+    const cmd = process.platform === "win32" ? "npm.cmd" : "npm"
+    return `${cmd} list --omit dev -a --json --long`
+  }
+
+  getDependenciesTree(): DependencyTree {
+    const npmListOutput = execSync(this.getPMCommand(), {
```

Comment:
For all areas using `execSync`, we have a function in `builder-util > util.ts` for this that includes debug logging. Can we utilize those to consolidate usage and leverage already-existing util code here?
https://github.com/electron-userland/electron-builder/blob/74d98d8aaecf1f6ee2fbd7b639972d3eab1cd0c5/packages/builder-util/src/util.ts#L73
https://github.com/electron-userland/electron-builder/blob/74d98d8aaecf1f6ee2fbd7b639972d3eab1cd0c5/packages/builder-util/src/util.ts#L165

Note, this would require minor signature change to the other function:
`getPMCommand(): { cmd: string, args: string[])`

Reply:
I'v changed it to use exec from util module.